src/design-guidelines/action-buttons/examples/icon-color.tsx

```diff
@@ -0,0 +1,83 @@
+    tag: 'limel-example-action-buttons-icon-color',
+    shadow: true,
+    styleUrl: 'action-buttons.scss',
+})
+export class ActionButtonsIconColorExample {
+    public render() {
+        return [
+            <div class="action-buttons-examples">
+                <div class="example">
+                    <div class="fake-dialog-container shows-full-dialog">
+                        <div class="fake-dialog">
+                            <limel-header
+                                icon="ask_question"
+                                heading="Save changes?"
+                                class="save"
+                            />
+                            <p>
+                                You have unsaved change. Do you want to save
+                                them before leaving this page?
+                            </p>
+                            <div class="action-bar">
+                                <limel-button
+                                    label="Back to editing"
+                                    class="button back primary--neutral justify-left"
+                                    icon="left_arrow"
+                                />
+                                <limel-button
+                                    label="Discard"
+                                    class="button discard primary--caution"
+                                    icon="cancel"
+                                    primary={true}
+                                />
+                                <limel-button
+                                    label="Save"
+                                    class="button update"
+                                    icon="ok"
+                                    primary={true}
+                                />
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="example">
+                    <div class="fake-dialog-container shows-full-dialog">
+                        <div class="fake-dialog">
+                            <limel-header
+                                icon="delete_message"
+                                heading="Delete selected items?"
+                                class="delete"
+                            />
+                            <p>
+                                You are about to delete 23 items. This is a
+                                permanent action and cannot be undone!
+                            </p>
+                            <div class="action-bar">
+                                <limel-button
+                                    label="Back to selection"
+                                    class="button primary--neutral justify-left"
+                                    icon="left_arrow"
+                                />
+                                <limel-button
+                                    label="Don't delete"
```

Comment:
This choice is non-obvious to me. With some thought, I understand it, but it's not optimal.

"Continue without deleting" is long, but at least explains the action.

Reply:
I think the length is the problem itself indeed. 
I have seen labels like this many times in Mac OS context. For instance:
![image](https://user-images.githubusercontent.com/35954987/126750005-61a23da5-f624-414e-8848-3f1798683436.png)
So it should be fine.

Reply:
Ah… ok, the "Save" vs "Don't Save" makes more sense to me, because I can imagine the context, and thus imagine the difference between the actions: I'm closing a modified document, *Save* will save and close, *Don't Save* will discard my changes and close, and *Cancel* will take me back to the document, not close it.

I'm having trouble imagining the context for your example though:

![image](https://user-images.githubusercontent.com/154725/126755229-78278148-e6fb-461e-9414-3997a4f800c1.png)

It seems I've probably been on some kind of list or table, and selected multiple items, then pressed some button to delete the selected items. If I press "Back to selection", that's obviously analogous to "Cancel" in your screenshot; I go back to the list or table I came from, and nothing is deleted. If I press "Delete", I obviously delete the items, but where do I end up after that? Wouldn't I normally be back at the list or table? But if so, then what's the difference between "Back to selection" and "Don't delete"? Both would _not delete the items_, obviously, but both would also have me back at the list or table, no?

I understand that this is just a made up example, and to many, the context won't matter, and sure, I understand what the example is trying to say anyway, but the fact that I can't imagine a context that makes sense gives me the feeling that maybe I've missed or misunderstood something. I don't get a sense of confidence, but start to doubt either myself or the reliability of the documentation (probably both, with somewhat alternating intensity). While I know I'm probably odd, I also assume I'm not unique 😆

Reply:
I totally see your point! ;)

It's quite hard to illustrate the entire interaction flow chart in one shot though. 

_Step 1 > step 2 > modal with action buttons > different outcomes_


I hope the reader can imagine that there was a step before this modal where the imaginary user selected 23 items (while they could select more or less) like in a table view or list view. Then they did an action, for example clicked on [Delete & Close] or [Continue the cleanup process…] or whatever action that "leaves" this current page and continues, permanently removing their ability to go back and change their mind. Now they are asked if they're sure that this is the outcome they want. They maybe accidentally chose 23 items, maybe 20 items were selected in a page before, and 3 in the visible current page (imagine a table with pagination). 

I just had a hard time to find a better example. But it'd be nice to find ways of explaining and clarifying the situation that lead to this modal. If you can find a better scenario please suggest :) 

Otherwise, I think the example is good enough for the purpose of this guide. At the end of the day, these guidelines are for inspiration, and to sync conventions in our different products. Developers shouldn't be left alone to design as well as code. When and how a confirmation modal should be shown, and with how many buttons and which button labels is subject to a design decision, and hopefully there is a designer supporting them for each case.

Reply:
👍

It's absolutely good enough. Let's get this baby merged! 🙂